### DISCUSSION TITLE
Santoker R200 Bluetooth

### DISCUSSION THREAD

**Discussion by acx4** [artisan-roaster-scope/artisan#1089] [General]
Hi I have a Santoker R200 Bluetooth (There is a model with Bluetooth and without)  which is working on the Santoker app, it's my sample roaster and I would like to use Artisan for both the sample and the production roaster. The Santoker App works on/over  Bluetooth in Windows 10 and Linux Ubuntu 22.04 LTS, but Artisan does not. I also have a smaller Q5 2.0 which that does work via WiFi (private network by Santoker module called Santoker 5) on both the Santoker App and Artisan.  Has anyone have any experience with Artisan getting data over Bluetooth on the Santoker machines?

**Comment by MAKOMO** [comment]
Artisan is not supporting any Santoker machine over Bluetooth currently. Please submit a PR with the missing code to make this happen. Make sure it works cross-platform!
